src/main/scala/org/hammerlab/guacamole/variants/CalledAllele.scala

```diff
@@ -41,50 +38,4 @@ case class CalledAllele(sampleName: String,
                         length: Int = 1) extends ReferenceVariant {
   val end: Long = start + 1L
 
-}
-
-class CalledAlleleSerializer
-    extends Serializer[CalledAllele]
-    with HasAlleleSerializer
-    with HasGenotypeEvidenceSerializer {
-
-  def write(kryo: Kryo, output: Output, obj: CalledAllele) = {
-    output.writeString(obj.sampleName)
-    output.writeString(obj.referenceContig)
-    output.writeLong(obj.start, true)
-    alleleSerializer.write(kryo, output, obj.allele)
-    alleleEvidenceSerializer.write(kryo, output, obj.evidence)
-
-    output.writeBoolean(obj.rsID.isDefined)
-    obj.rsID.foreach(output.writeInt(_, true))
-    output.writeInt(obj.length, true)
-  }
-
-  def read(kryo: Kryo, input: Input, klass: Class[CalledAllele]): CalledAllele = {
-
-    val sampleName: String = input.readString()
-    val referenceContig: String = input.readString()
-    val start: Long = input.readLong(true)
-    val allele = alleleSerializer.read(kryo, input, classOf[Allele])
-    val evidence = alleleEvidenceSerializer.read(kryo, input, classOf[AlleleEvidence])
-    val hasRsID = input.readBoolean()
-    val rsId =
-      if (hasRsID)
-        Some(input.readInt(true))
-      else
-        None
-    val length: Int = input.readInt(true)
-
-    CalledAllele(
-      sampleName,
-      referenceContig,
-      start,
-      allele,
-      evidence,
-      rsId,
-      length
-    )
-
-  }
-}
-
+}
```

Comment:
fwiw you can have intellij add newlines to ends of files on save:

![](http://cl.ly/image/3R0P2p1y1E2A/Screen%20Shot%202015-08-21%20at%204.27.58%20PM.png)